### PR TITLE
LPS-152817 AssetDisplayPagesItemSelectorView

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
cc: @DiegoHu97 @lfbesada

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by BarbaraCabrera** [comment]
c:test:sf

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
ci:reevaluate:1327401_4853

**Comment by Tim-Cao** [comment]
Hi @BarbaraCabrera, there is a regression issue. Would you mind having a look at?
Cannot choose a page where set Asset Publisher as default for page in display page selector.
**Steps to reproduce**

1. Add an Asset Publisher to a widget page
2. Set the Assset Publisher as default for page in configuration of Asset Publisher
3. Add a web content
4. Select the widget page as specific display page

**Actual Results:**
The select page modal doesn't close.
image

**Expected Results:**
The selected page should be shown in display page selector.

**Comment by BarbaraCabrera** [comment]
Thank you @Tim-Cao, I'll take a look.

> Hi @BarbaraCabrera, there is a regression issue. Would you mind having a look at? Cannot choose a page where set Asset Publisher as default for page in display page selector. **Steps to reproduce**
> 
> 1. Add an Asset Publisher to a widget page
> 2. Set the Assset Publisher as default for page in configuration of Asset Publisher
> 3. Add a web content
> 4. Select the widget page as specific display page
> 
> **Actual Results:** The select page modal doesn't close. image
> 
> **Expected Results:** The selected page should be shown in display page selector.

File: modules/apps/item-selector/item-selector-web/src/main/resources/META-INF/resources/view_item_selector_view_descriptor.jsp [hunk 1]
```diff
@@ -176,7 +176,7 @@ SearchContainer<Object> searchContainer = itemSelectorViewDescriptorRendererDisp
 							%>
 
 							<span class="text-default">
-								<liferay-ui:message arguments="<%= LanguageUtil.getTimeDescription(request, System.currentTimeMillis() - modifiedDate.getTime(), true) %>" key="modified-x-ago" />
+								<liferay-ui:message arguments="<%= LanguageUtil.getTimeDescription(request, System.currentTimeMillis() - modifiedDate.getTime(), true) %>" key="x-ago" />
```
Comment (ealonso) [inline modules/apps/item-selector/item-selector-web/src/main/resources/META-INF/resources/view_item_selector_view_descriptor.jsp:179]: @BarbaraCabrera @ruben-pulido @lfbesada 

This change should be send to lima team

**Comment by BarbaraCabrera** [comment]
ci:test:relevant

**Comment by ozysouza** [comment]
Hi @BarbaraCabrera, the unique failure is not related to your changes. :)

**Comment by DiegoHu97** [comment]
Resent here: https://github.com/liferay-echo/liferay-portal/pull/9061

**Pull request closed without merging**